asv_bench/asv.conf.json

```diff
@@ -41,7 +41,10 @@
         "sqlalchemy": [],
         "scipy": [],
         "numexpr": [],
-        "pytables": [],
+        "tables": [],
```

Comment:
@RafalSkolasinski Looks like Anaconda changed the package name - it's on PyPI as ```tables```: https://pypi.python.org/pypi/tables . I didn't want to force people to use ```conda``` or ```virtualenv```, but I don't see an easy way to support both due to this name change.